internal/pkg/controllers/spod/configmap_controller.go

```diff
@@ -61,9 +62,15 @@ func (r *ReconcileSPOd) Reconcile(request reconcile.Request) (reconcile.Result,
 		}
 		return reconcile.Result{}, fmt.Errorf("error getting spod configuration: %w", err)
 	}
+
+	namespace, err := config.GetOperatorNamespace()
+	if err != nil {
+		return reconcile.Result{}, errors.Unwrap(err)
```

Comment:
Why use `.Unwrap` here? That will lose the error:

```
$ cat err.go 
package main

import (
        "errors"
        "fmt"
)

func main() {
        var ErrPodNamespaceEnvNotFound = errors.New("the env variable MY_POD_NAMESPACE hasn't been set")
        unwrapped := errors.Unwrap(ErrPodNamespaceEnvNotFound)
        fmt.Printf("%T\n", unwrapped)
}                                                                                                                                                                                                                                                     
 $ go run err.go
<nil>
```

Reply:
I did that because of `wrapcheck` error from golang-lint

![image](https://user-images.githubusercontent.com/172697/105751068-84163f80-5f13-11eb-972b-653c776647ab.png)

Reply:
Probably revisit whether `wrapcheck` makes sense.

Reply:
That check is asking for the opposite, use `errors.Wrap` to add more context to the error

Reply:
@cmurphy Thanks, my ignorance. I will address this.